Internal Server Error: /main/
Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 875, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
TypeError: 'Renginys' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'Renginys' object has no attribute 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    current = current[int(bit)]
                      ~~~^^^^^
ValueError: invalid literal for int() with base 10: 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\mainPage\views.py", line 29, in main_page
    return render(request, 'main.html', context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\defaulttags.py", line 242, in render
    nodelist.append(node.render_annotated(context))
                    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 731, in resolve
    arg_vals.append(arg.resolve(context))
                    ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 842, in resolve
    value = self._resolve_lookup(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 898, in _resolve_lookup
    raise VariableDoesNotExist(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
    )  # missing attribute
    ^
django.template.base.VariableDoesNotExist: Failed lookup for key [data] in <Renginys: TestDate (Ansamblis2)>
"GET /main/ HTTP/1.1" 500 306
Internal Server Error: /main/
Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 875, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
TypeError: 'Renginys' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'Renginys' object has no attribute 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    current = current[int(bit)]
                      ~~~^^^^^
ValueError: invalid literal for int() with base 10: 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\mainPage\views.py", line 29, in main_page
    return render(request, 'main.html', context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\defaulttags.py", line 242, in render
    nodelist.append(node.render_annotated(context))
                    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 731, in resolve
    arg_vals.append(arg.resolve(context))
                    ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 842, in resolve
    value = self._resolve_lookup(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 898, in _resolve_lookup
    raise VariableDoesNotExist(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
    )  # missing attribute
    ^
django.template.base.VariableDoesNotExist: Failed lookup for key [data] in <Renginys: TestDate (Ansamblis2)>
"GET /main/ HTTP/1.1" 500 306
Internal Server Error: /main/
Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 875, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
TypeError: 'Renginys' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'Renginys' object has no attribute 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    current = current[int(bit)]
                      ~~~^^^^^
ValueError: invalid literal for int() with base 10: 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\mainPage\views.py", line 29, in main_page
    return render(request, 'main.html', context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\defaulttags.py", line 242, in render
    nodelist.append(node.render_annotated(context))
                    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 731, in resolve
    arg_vals.append(arg.resolve(context))
                    ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 842, in resolve
    value = self._resolve_lookup(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 898, in _resolve_lookup
    raise VariableDoesNotExist(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
    )  # missing attribute
    ^
django.template.base.VariableDoesNotExist: Failed lookup for key [data] in <Renginys: TestDate (Ansamblis2)>
"GET /main/ HTTP/1.1" 500 306
Internal Server Error: /main/
Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 875, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
TypeError: 'Renginys' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'Renginys' object has no attribute 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    current = current[int(bit)]
                      ~~~^^^^^
ValueError: invalid literal for int() with base 10: 'data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\mainPage\views.py", line 29, in main_page
    return render(request, 'main.html', context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\defaulttags.py", line 242, in render
    nodelist.append(node.render_annotated(context))
                    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 1059, in render
    output = self.filter_expression.resolve(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 731, in resolve
    arg_vals.append(arg.resolve(context))
                    ~~~~~~~~~~~^^^^^^^^^
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 842, in resolve
    value = self._resolve_lookup(context)
  File "C:\KTU_PS\Bakalauras\FolklorasProj\venv1\Lib\site-packages\django\template\base.py", line 898, in _resolve_lookup
    raise VariableDoesNotExist(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<2 lines>...
    )  # missing attribute
    ^
django.template.base.VariableDoesNotExist: Failed lookup for key [data] in <Renginys: TestDate (Ansamblis2)>
"GET /main/ HTTP/1.1" 500 306
"GET /kuriniai/2/edit/ HTTP/1.1" 500 306
